src/queue.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+#
+# queue.py
+# Created by Stephen Brimhall on 1/23/17
+# Copyright (c) SST CTF and Stephen Brimhall. All Rights Reserved.
+#
+# Implements a simple threadsafe queue object.
+#
+
+# Begin system imports
+from threading import RLock as Lock;
+# End system imports
+
+# Queue class
+
+class Queue(object):
```

Comment:
Pretty much. I am just using it as a way to organize threadsafe lists for now, I may implement some safer way of managing it later but for now good practice will do.